src/syzygy/tbprobe.cpp

```diff
@@ -726,8 +726,6 @@ class TBTables {
     typedef std::tuple<Key, TBTable<WDL>*, TBTable<DTZ>*> Entry;
 
 #if defined(ANTI)
```

Comment:
Should I drop more than what's indicated in this PR?

Reply:
So there were branches for anti, atomic and standard. Atomic can use the same size as standard. (The branch for atomic was only nescessary initially due to "bad luck" with the material hash keys).

Or maybe you are referring to a version of this PR that was broken, before I force pushed over it.

Reply:
Ah, that makes perfect sense and now I understand this PR.  Thanks!